flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/SqlParseUtils.java

```diff
@@ -87,7 +87,11 @@ public static Map<String, String> extractMap(@Nullable SqlNodeList propList) {
         }
         return propList.getList().stream()
                 .map(p -> (SqlTableOption) p)
-                .collect(Collectors.toMap(k -> k.getKeyString(), SqlTableOption::getValueString));
+                .collect(
+                        Collectors.toMap(
+                                k -> k.getKeyString(),
+                                SqlTableOption::getValueString,
+                                (v1, v2) -> v2));
```

Comment:
Since by default java throws exception about duplicated keys...

Reply:
Should we add a WARN?